### PR TITLE
chore: add changelog doc

### PR DESCRIPTION
**Pull request by hhyasdf**
Pull Request Description
---

### Describe what this PR does / why we need it
Add changelog doc.

### Does this pull request fix one issue?

#2 

### Special notes for reviews

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -0,0 +1,16 @@
+# Rama Changelog
+All notable changes to this project will be documented in this file.
+
+## [v0.1.0] 2021-06-18
```
Comment (mars1024) [inline CHANGELOG.md:4]: a timestamp is needless here, git will help us for the history

File: CHANGELOG.md [hunk 2]
```diff
@@ -0,0 +1,16 @@
...
+### New features
+- First release version
```
Comment (mars1024) [inline CHANGELOG.md:6]: is this a new feature?

File: CHANGELOG.md [hunk 3]
```diff
@@ -0,0 +1,16 @@
...
+- Support overlay (vxlan) network
+- Support hybrid overlay/underlay container network
+- Full support for ipv4/ipv6 dual-stack
+- Node need only one physical nic if container network is in the same vlan with node network
```
Comment (mars1024) [inline CHANGELOG.md:10]: also i don't think this is a feature, maybe it's an instruction on a feature

File: CHANGELOG.md [hunk 4]
```diff
@@ -0,0 +1,16 @@
...
+- Non-zero-netId subnet and zero-netId subnet can be on the same node
```
Comment (mars1024) [inline CHANGELOG.md:11]: ditto

File: CHANGELOG.md [hunk 5]
```diff
@@ -0,0 +1,16 @@
...
+- Webhook configuration can be managed by an independent yaml
```
Comment (mars1024) [inline CHANGELOG.md:12]: ditto

File: CHANGELOG.md [hunk 6]
```diff
@@ -0,0 +1,16 @@
...
+
+### Fixed Issues
+- Remove overlay logs for underlay-only schema
```
Comment (mars1024) [inline CHANGELOG.md:15]: nit: schema --> mode

File: CHANGELOG.md (hunk 1 above)
Comment (hhyasdf) [reply CHANGELOG.md]: removed

File: CHANGELOG.md (hunk 2 above)
Comment (hhyasdf) [reply CHANGELOG.md]: removed

File: CHANGELOG.md (hunk 3 above)
Comment (hhyasdf) [reply CHANGELOG.md:11]: change to improvements

File: CHANGELOG.md (hunk 4 above)
Comment (hhyasdf) [reply CHANGELOG.md:12]: change to improvements

File: CHANGELOG.md (hunk 5 above)
Comment (hhyasdf) [reply CHANGELOG.md:13]: change to improvements

File: CHANGELOG.md (hunk 6 above)
Comment (hhyasdf) [reply CHANGELOG.md]: done~

**Review by mars1024 — APPROVED** [review]
/lgtm
/approve

**Pull request merged**